crates/bdk/src/wallet/signer.rs

```diff
@@ -849,19 +903,26 @@ impl ComputeSighash for Legacy {
     }
 }
 
-fn p2wpkh_script_code(script: &Script) -> Script {
+fn p2wpkh_script_code(script: &ScriptBuf) -> ScriptBuf {
```

Comment:
We have `ScriptBuf::p2wpkh_script_code` now (I actually took the code from bdk when adding it :)